mungegithub/github/github.go

```diff
@@ -1786,6 +1786,26 @@ func (obj *MungeObject) AddAssignee(owner string) error {
 	return nil
 }
 
+// IsAssignee will determine if the owner is a valid assignee in this repo
+func (obj *MungeObject) IsAssignee(owner string) (bool, error) {
+	_, response, err := obj.config.client.Issues.IsAssignee(
+		context.Background(),
+		obj.Org(),
+		obj.Project(),
+		owner,
+	)
+	if err != nil {
+		glog.Errorf("Error determining if user %v is a valid assignee: %v", owner, err)
+		return false, err
+	}
+	// For this method, 404 is marked as not an error
```

Comment:
Are we sure that on 404, err is nil? When I was adding the oauth scope suggestion, on 403s errors were non-nil.